src/test/java/org/jvnet/hudson/plugins/seleniumhtmlreport/SeleniumHtmlReportTest.java

```diff
@@ -0,0 +1,67 @@
+package org.jvnet.hudson.plugins.seleniumhtmlreport;
+
+import hudson.model.Result;
+import jenkins.tasks.SimpleBuildStep;
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.jenkinsci.plugins.workflow.steps.CoreStep;
+import org.jenkinsci.plugins.workflow.steps.StepConfigTester;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runners.model.Statement;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.RestartableJenkinsRule;
+
+import java.nio.file.Paths;
+
+import static org.junit.Assert.*;
+
+public class SeleniumHtmlReportTest {
+
+    @Rule
+    public RestartableJenkinsRule story
+            = new RestartableJenkinsRule();
```

Comment:
Just use `JenkinsRule` unless you are actually testing Jenkins restarts (irrelevant in this plugin).